src/main/java/com/google/sps/model/AuthenticationVerifier.java

```diff
@@ -14,15 +14,20 @@
 
 package com.google.sps.model;
 
+import com.google.sps.utility.KeyProvider;
 import java.io.IOException;
 import java.security.GeneralSecurityException;
 
 /** Contract for verifying user with Google Sign in API */
 public interface AuthenticationVerifier {
```

Comment:
Just a HU that this is already in master, so you probably need to merge master into `mvp/calendar-panel`, and then merge `mvp/calendar-panel` into this branch

Reply:
`Resolved`